Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+        public static void Demo()
+        {
+            const int maxEntries = 3;
+
+            for (int i = 1; i < maxEntries; i++)
+            {
+                Console.Clear();
+
+                RegisterNewPersonData(i);
+            }
+        }
+
+        static void RegisterNewPersonData(int entry)
+        {
+            Console.WriteLine($"Enter the following information for person #{entry}.");
+
+            string firstName = Checks.PromptString("Firstname: ");
+
+            string lastName = Checks.PromptString("Lastname: ");
+
+            int age = Checks.PromptInt("Age:");
+
+            float weight = Checks.PromptFloat("Weight in kilograms:");
+
+            float height = Checks.PromptFloat("Height in meters:");
+
+            var bmi = Checks.CalculateBmi(weight, height);
```

Comment:
You CAN do that but you give control from Lesson3 to Checks to Lesson3 which is unneccessary. Use the functions below directly! :)
Also consider the following:
RegisterAge() for example reads in an int value but that has nothing to do with the age. The age is how you interpret this value.
Your function does an entirely different thing than you describe it with that name. If you want to do it properly you can create a new (private?) function which reads in the int and a new public function that handles the age aspect.
This also makes sense when looking at the homework 4 task (but don't try to solve 3 and 4 at the same time! :D)

Reply:
Thanks for your time in doing this review. 

I quite don't follow on:
> Use the functions below directly! :)

Do you mean that I should use the methods in lesson3.cs instead of the checks? If so I don't understand why checks.cs is needed at all

Reply:
Yes you should not use Checks.X().
Those functions are wrapper functions that were defined by Kai so you could write your own functions as you want. You can name them as much as you like and all you need to do is to set them up in the Checks.cs file. :)

Reply:
I think I got it :)

Reply:
I think I got it =)